### PR TITLE
srv update

### PR DESCRIPTION
**Pull request by grahamc**
##### Description

tested

##### Checklist

- [ ] Tested changes against a test repository
- [ ] Added or updated relevant documentation (leave unchecked if not applicable)
- [ ] (If this PR is for a release) Updated README to point to the new tag (leave unchecked if not applicable)

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ inputs:
     default: https://cache.nixos.org
   diagnostic-endpoint:
     description: "Diagnostic endpoint url where diagnostics and performance data is sent. To disable set this to an empty string."
-    default: "https://install.determinate.systems/magic-nix-cache/perf"
+    default: "-"
```
Comment (colemickens) [inline action.yml:19]: Why `"-"` over `""`? Does detsys-ts allow the latter as well, at least?

Comment (grahamc) [reply action.yml:19]: Users can already set the value to "" to disable it, so if we make the default "" and change how to disable it ... that'd be crummy.

File: src/index.ts [hunk 2]
```diff
@@ -43,6 +43,7 @@ class MagicNixCacheAction extends DetSysAction {
       fetchStyle: "gh-env-style",
       idsProjectName: "magic-nix-cache-closure",
       requireNix: "warn",
+      diagnosticsSuffix: "perf",
```
Comment (colemickens) [inline src/index.ts:46]: I noticed the update-flake-lock doesn't have this set. And `perf`/`telemetry`, not sure the name scheming? Of course, maybe this is just temporary for an initial run through CI checks?

Comment (grahamc) [reply src/index.ts:46]: The projects don't have a consistent name for them for historical reasons. This suffix matches what was in the action.yml.

**Review by colemickens — APPROVED** [review]

**Pull request merged**